runbotics/runbotics-desktop/src/action/folder/folder.action-handler.ts

```diff
@@ -39,10 +39,30 @@ export default class FolderActionHandler extends StatelessActionHandler {
         }
     }
 
+    async displayFiles(input: FolderDisplayFilesActionInput): Promise<FolderDisplayFilesActionOutput | null> {
+        const { name, path } = input;
+        
+        const folderPath = path ?
+            `${path}${pathPackage.sep}${name}` :
+            `${this.serverConfigService.tempFolderPath}${pathPackage.sep}${name}`;
+
+        try {
+            return fs.readdirSync(folderPath);
```

Comment:
what is the example result?

Reply:
returns array of strings, eg.
['folder', 'folder2', 'fileName1', 'file2']